treasure-keeper.cabal

```diff
@@ -25,11 +25,17 @@ library
 
   exposed-modules:     TreasureKeeper
```

Comment:
Maybe we can remove `TreasureKeeper` module?

Reply:
🤔 For now yes. But maybe it's better to have this `executeTreasure` function there?

Reply:
Hmm, this is a good idea as well! Can can